pyspark-iris/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/context.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2020 QuantumBlack Visual Analytics Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+# OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, AND
+# NONINFRINGEMENT. IN NO EVENT WILL THE LICENSOR OR OTHER CONTRIBUTORS
+# BE LIABLE FOR ANY CLAIM, DAMAGES, OR OTHER LIABILITY, WHETHER IN AN
+# ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF, OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+#
+# The QuantumBlack Visual Analytics Limited ("QuantumBlack") name and logo
+# (either separately or in combination, "QuantumBlack Trademarks") are
+# trademarks of QuantumBlack. The License does not grant you any right or
+# license to the QuantumBlack Trademarks. You may not use the QuantumBlack
+# Trademarks or any confusingly similar mark as a trademark for your product,
+# or use the QuantumBlack Trademarks in any other manner that might cause
+# confusion in the marketplace, including but not limited to in advertising,
+# on websites, or on software.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Entry point for running a Kedro pipeline as a Python package."""
+from pathlib import Path
+from typing import Any, Dict, Union
+
+from kedro.framework.context import KedroContext
+
+from pyspark import SparkConf
+from pyspark.sql import SparkSession
+
+
+class ProjectContext(KedroContext):
```

Comment:
I think it would be a good idea to add a small comment about this being a custom context which is needed to setup the spark session etc. I assume people will look at the starters for inspiration so explanations about why a class exists will help them understand if they need it or not.

Reply:
I added some explanation. Could you take a look and let me know if it looks okay?

Reply:
Short and sweet, I like it! 😄